### ISSUE TITLE
The pretrained network for Deeplabv2

### ISSUE THREAD

**Issue by Zhengyang1995** [qjadud1994/DRS#16]
As the pretrained network you provide for deeplab v2, https://drive.google.com/file/d/1Wj8Maj9KGQgwtDfvIp8FChsdAIgDvliT/view, the name of it is deeplabv1_resnet101-coco.pth, is it correct?

**Comment by qjadud1994** [comment]
Yes, it is correct.

As we mentioned in [here](https://github.com/qjadud1994/DRS#training-the-deeplab-v2-using-pseudo-labels), we strictly followed the [deeplab-pytorch implementation](https://github.com/kazuto1011/deeplab-pytorch) for the semantic segmentation network learning part.

Please check [here](https://github.com/kazuto1011/deeplab-pytorch#download-pre-trained-caffemodels).
